gamedb.yaml

```diff
@@ -4738,6 +4743,12 @@
   name: Tomb Raider III
   compat_tool: Proton-6.1-GE-2
 
+- name: 'The Outer Worlds: Spacer's Choice Edition'
```

Comment:
This needs to be using double quotes because of the `'` in the name.

Reply:
Whoops. Fixed in the latest commit.

Reply:
Awesome, thank you!